README.md

```diff
@@ -1 +1,100 @@
+
+https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup
+
+[Andersson 2003](https://www.ncbi.nlm.nih.gov/pubmed/14568458) J.L.R. Andersson, S. Skare, J. Ashburner. How to correct susceptibility distortions in spin-echo echo-planar images: application to diffusion tensor imaging. NeuroImage, 20(2):870-888, 2003.
+
+[Smith 2004](https://www.ncbi.nlm.nih.gov/pubmed/15501092) S.M. Smith, M. Jenkinson, M.W. Woolrich, C.F. Beckmann, T.E.J. Behrens, H. Johansen-Berg, P.R. Bannister, M. De Luca, I. Drobnjak, D.E. Flitney, R. Niazy, J. Saunders, J. Vickers, Y. Zhang, N. De Stefano, J.M. Brady, and P.M. Matthews. Advances in functional and structural MR image analysis and implementation as FSL. NeuroImage, 23(S1):208-219, 2004.
+
+[Graham 2017](https://www.ncbi.nlm.nih.gov/pubmed/28968429) M.S. Graham, I. Drobnjak, H. Zhang. Quantitative assessment of the susceptibility artefact and its interaction with motion in diffusion MRI. PLoS ONE, 12(10), 2017.
+
+## Gear usage
+This gear currently uses two images that are sampled with different (ideally opposite) phase-encoding directions and calculates the topup fieldmap.  This gear optionally performs a topup correction on the two input images, and can also perform topup correction on two additional images.
+
+The gear allows you to choose a number of optional outputs besides the standard topup field approximation.
+The gear always runs topup with the *--out*, *--fout*, and *--iout* options.
+
+### Inputs
+
+#### Required
+* **image_1**: the first image to use in the topup calculation.
+* **image_2**: the second image to use in the topup calculation.
+* **acquisition_parameters**: the input to the [--datain](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/TopupUsersGuide#A--datain) option for topup, a text file with PE directions/times.  In this text file, row 1 corresponds to **image_1**, and row 2 corresponds to **image_2**.
+
+**image_1** and **image_2** can be either 3D or 4D images.  If they are 3D, then topup is estimated, and by default these two are corrected.  If they are 4D, then the first volume from each acquisition is used for the topup calculation.  Then, topup correction is applied to the full 4D image, unless **topup_only** is selected from the config file.
+
+
+#### Optional
+* **apply_to_1**: A third image to apply the topup correction calculated from **image_1** and **image_2**.  This image must have the same PE direction as **image_1**
+* **apply_to_2**: A fourth image to apply the topup correction calculated from **image_1** and **image_2**.  This image must have the same PE direction as **image_2**
+* **config_file**: a config file to pass into topup to use with the [--config](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/TopupUsersGuide#Configuration_files) option.
```

Comment:
I like how you referenced the documentation here.

Would it be too much to refer to documentation where each topup option is used?

Reply:
The config file is really the only option.  I suppose there are some output options, but some of them aren't documented.  I'm assuming that people kinda know what they're doing with this.